salesagility/SuiteCRM
# Fix #9665 - Set unique id for "Reset module" button in studio

## Description





- resolve [#9665](https://github.com/salesagility/SuiteCRM/issues/9665)

As described in issue, the button _Reset module_ has properties `id` y `name` _exportBtn_, that coincide with another study button.

This PR set an unique and appropriate `id` and `name`  for _Reset module_ button
## Motivation and Context

In some developments we have to apply, through the general CSS, some specific rule for one of these buttons and we cannot do it without affecting the other button, which shares the same id.

## How To Test This

1. Open studio
2. Open any module
3. With browser inspector look at the _Reset module_ button and check that the button has its own id and name

## Types of changes


### Final checklist
